terraform/modules/cluster/gke-beta/main.tf

```diff
@@ -16,6 +16,7 @@
 locals {
   gke_master_version   = var.gke_version != null ? var.gke_version : data.google_container_engine_versions.gkeversion.latest_master_version
   node_service_account = var.node_service_account == null ? data.google_compute_default_service_account.account.email : var.node_service_account
+  gke_endpoint_value   = var.gke_endpoint == null ? "https://container.googleapis.com/" : var.gke_endpoint
```

Comment:
Why not set default to `https://container.googleapis.com/` instead of `null`?

Reply:
> Why not set default to `https://container.googleapis.com/` instead of `null`?

It makes it easy to handle null values. Even if the user set this value to `null` in terraform.tfvars we will still do the right thing.

Reply:
use `nullable = false` in the `variable` block. if you do that, the value will be the default value when the use sets it to `null`